texttospeech/api/README

```diff
@@ -0,0 +1,59 @@
+# .NET Cloud TextToSpeech API Samples
+
+A collection of samples that demonstrate how to call the 
+[Google Cloud TextToSpeech API](https://cloud.google.com/text-to-speech/) from C#.
+
+This sample requires [.NET Core 2.0](
+    https://www.microsoft.com/net/core) or later.  That means using
+[Visual Studio 2017](
+
+    https://www.visualstudio.com/), or the command line.
+## Build and Run
+
+1.  **Follow the instructions in the [root README](../../README.md)**.
+
+4.  Enable APIs for your project.
+    [Click here](https://console.cloud.google.com/flows/enableapi?apiid=texttospeech.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud TextToSpeech API.
+
+9.  From a Powershell command line, run the QuickStart sample:
+    ```
+    PS C:\...\dotnet-docs-samples\texttospeech\api\Quickstart> dotnet restore
+    PS C:\...\dotnet-docs-samples\texttospeech\api\Quickstart> dotnet run
+    # Creates 'sample.mp3' file in project folder
+    ```
+
+10. And run TextToSpeech to synthesize text to audio.
+    ```
```

Comment:
Before finalizing, just be sure to re-run the CLI and copy/paste the Help output here – incase it changes :)

Reply:
Done